src/core/best-practices.md

```diff
@@ -155,3 +155,58 @@ You can use our [Annotation Plugin](/core/software/pupil-capture/#annotations) t
 the recording on desired events, such as trigger events. Annotations can be sent via the keyboard or programmatically. 
 [This script](https://github.com/pupil-labs/pupil-helpers/blob/master/python/remote_annotations.py) demonstrates how you 
 can send remote annotations over the network.
+
+## Pupillometry
+
```

Comment:
@N-M-T do we want to ever define the word `pupillometry` and then link out to wikipedia or other source for longer discussion. I feel like many people new to eye tracking might not even know what the word means.

Reply:
I've tried to make this more accessible

Reply:
I like the summary